ai/executors/motion_executor.py

```diff
@@ -251,10 +252,9 @@ def reset(self):
 def get_control_setting(is_sim: bool):
 
     if is_sim:
-        translation = {"kp": 0.8, "ki": 0.01, "kd": 0, "antiwindup": 20, "deadzone": 0, "sensibility": 0}
-        rotation = {"kp": 2, "ki": 0, "kd": 0.1, "antiwindup": 0, "deadzone": 0, "sensibility": 0}
+        translation = {"kp": 0.8, "ki": 0, "kd": 1, "antiwindup": 20, "deadzone": 0, "sensibility": 0}
```

Comment:
Nevermind what i might have said before, I think it could be nice in the config file. Just make sure we keep some default values somewhere in case.